pysgrid/sgrid.py

```diff
@@ -514,14 +525,15 @@ def save_as_netcdf(self, filepath):
             # populate variables with data
             grid_time[:] = self.grid_times[:]
             grid_center_lon[:] = self.centers[..., 0]
-            grid_center_lat[:] = self.centers[..., 1] 
-            grid_node_lon[:] = self.nodes[..., 0]
-            grid_node_lat[:] = self.nodes[..., 1]
+            grid_center_lat[:] = self.centers[..., 1]
+            if self.nodes is not None: 
+                grid_node_lon[:] = self.nodes[..., 0]
+                grid_node_lat[:] = self.nodes[..., 1]
```

Comment:
> I like simpler code too; it simplifies development, and reduces maintenance and technical debt burdens.

Yep!

>  I don't have much desire to 'cowboy' SGrid3D's removal from master before a formal decision is made.

I do not see placing it on a branch as " cowboy SGrid3D's removal", in fact that is the best Open Source way to discuss an idea.  Not in master...